bin/generate_charts.py

```diff
@@ -372,19 +380,14 @@ def generate_charts(files):
     if args.output != '':
         output = path.join(args.output, '')
 
-    # Get list of directories
-    for (dirpath, dirnames, filenames) in walk(args.results_dir):
-        for file in filenames:
-            m = re.search(
```

Comment:
This regex search is being abandoned, whats the implication of this?

Reply:
This script has a restriction on the filenames it parses (not sure why). This format is not what OMB outputs, so seems like an unnecessary restriction. My opinion is to parse all valid json files recursively in the input directory and rather rely on the contents in the file to define uniqueness of a benchmark.

name = {workload name}-{driver-name}-{filename}

All 3 we can get from the file contents and its name and we don't have to pass this custom format to the `benchmark` command.

(Also see the other response to @tmgstevens comment)